azurerm/resource_arm_kubernetes_cluster.go

```diff
@@ -1335,6 +1445,39 @@ func flattenKubernetesClusterNetworkProfile(profile *containerservice.NetworkPro
 		values["load_balancer_sku"] = string(profile.LoadBalancerSku)
 	}
 
+	lbProfiles := make([]interface{}, 0)
+
+	if profile.LoadBalancerProfile != nil {
+		lb := make(map[string]interface{})
+
+		if profile.LoadBalancerProfile.ManagedOutboundIPs != nil {
+			lb["managed_outbound_ips"] = profile.LoadBalancerProfile.ManagedOutboundIPs.Count
+		}
+
+		if profile.LoadBalancerProfile.OutboundIPs != nil {
+			lb["outbound_ip_address_ids"] = profile.LoadBalancerProfile.OutboundIPs.PublicIPs
+		}
+
+		if profile.LoadBalancerProfile.OutboundIPPrefixes != nil {
+			lb["outbound_ip_prefixes_ids"] = profile.LoadBalancerProfile.OutboundIPPrefixes.PublicIPPrefixes
```

Comment:
(same here) since this returns a Pointer we need to de-reference this here; as such could we make this:

```suggestion
			if profile.LoadBalancerProfile.OutboundIPPrefixes.PublicIPPrefixes != nil {
				lb["outbound_ip_prefixes_ids"] = profile.LoadBalancerProfile.OutboundIPPrefixes.PublicIPPrefixes
			}
```